src/ComputationContainer/Test/IntegrationTest/ComputationToDbGateTest.hpp

```diff
@@ -1,14 +1,18 @@
 #include "gtest/gtest.h"
+#include <map>
 #include "nlohmann/json.hpp"
 #include "Client/ComputationToDbGate/Client.hpp"
```

Comment:
It is preferable to `include` the C++ standard library headers first.

Please refer following link:  
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

```suggestion
#include <map>

#include "gtest/gtest.h"
#include "nlohmann/json.hpp"

#include "Client/ComputationToDbGate/Client.hpp"
```

Reply:
I forgot that sorting is false in format. Thanks!